README.md

```diff
@@ -1 +1,142 @@
+```
+
+Then you can go to the hosts page in the Rancher UI and copy the docker run command to add the host to the system. Note that if you are running behind a firewall/NAT gateway (e.g. in Amazon) you will need to use the public address for the host as the target for your connection request. Depending on your gateway configuration, you may be able to discover this address with a curl request like:
+
+```
+curl -4 icanhazip.com
+```
+
+In addition to the host configuration, the system does not have user authentication enabled initially.  For the test enviornment, we recommend using the Local configuration option (Admin->Authentication) and creating an admin user with a password.
+
+And finally, we need to enable the secrets service (this is a plugin for Rancher available in the application catalog), and create and set the API key and the key and secret secrets. To load the Secrets plugin, select the Catalog->Library from the UI. Then search for Secrets and then "select" and "launch" the Secrets service.
+
+Now we can create and distribute secrets to our services.  Before we do that, we need to create a new API Key and Secret pair, which is available via the UI at API->Keys.  Create a secret pair called servo-secret, and then note the Key and Secret.
+
+*NOTE*: the Secret is only available in this display window and is not recorded in an accessible location after being displayed.
+
+Finally we can create the secrets that our servo will read. Select Infrastructure->Secrets from the UI.  Then select Add Secret, and create a secret called api_key, and add the API key value to this secret.  Do the same for a secret called api_secret, but install the API secret in this key.
+
+## Launch a Test App - opsani co-http
+
+The Opsani test app co-http can be used to provide a test target for the Opsani Optune engine within the rancher environment. And a docker_compose and rancher_compose file are provided in the example directory to support launching the application directly (the app can also be defined and deployed manually).
+
+To launch the app, select the Stacks->User from the UI, and then select Add Stack. Provide a name (e.g. http-test) for the stack, and then either copy and paste the contents of the docker_compose.yml and rancher_compose.yml into the respective sections of the UI, or browse to the examples directory and upload the file contents. Select Create to build out the test app.
+
+Note that the default service created is available on port 8090, and it will only be available on the local rancher host unless a port mapping, security group rule, or filrewall ACL is applied to make this port available in most cloud based VM environments (e.g. AWS).
+
+A quick check can be made to ensure that the application is running by running a curl command on the local Rancher host:
+
+curl http://localhost:8090/?call=http%3A%2F%2Fback%3A8080%2F%3Fbusy%3D40\&busy=10
+
+
+# GET GOING
+```
+python3 -m venv .venv
```

Comment:
This dir is not under version control, is that something that gets created during install?

Reply:
This should be strictly about development documentation. I'll change the wording to reflect that.

There is no need for a virtualenv outside of development.